(GH-1479) Remove experimental notice for inventory file v2 (#1490)
puppetlabs/bolt: documentation/inventory_file_v2.md

# Inventory file version 2

## Migrating to version 2

> Version 2 of the inventory file changes some terms and syntax. To convert to version 2, you must make these changes. This can be done manually by changing fields in the inventory file or automatically with a Bolt command.

Reviewer: ```suggestion
Version 2 of the inventory file changes some terms and syntax. To convert to version 2, you must make these changes. This can be done manually by changing fields in the inventory file or by running `bolt project migrate` at the root of your [Boltdir](bolt_project_directories.md#).
```
